app/assets/javascripts/components/story/ExpandedStory/ExpandedStoryTitle.js

```diff
@@ -0,0 +1,29 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+class ExpandedStoryTitle extends Component {
+
+  render() {
+    const { story, onEdit } = this.props;
+
+    return (
+      <div className="Story__section">
+        <div className="Story__section-title">
+          {I18n.translate('activerecord.attributes.story.title')}
+        </div>
+        <input
+          value={story._editing.title}
+          className="form-control input-sm"
+          onChange={(event) => onEdit({ title: event.target.value })}
+        />
+      </div>
+    );
+  };
+}
+
+ExpandedStoryTitle.propTypes = {
+  story: PropTypes.object.isRequired,
```

Comment:
Shouldn't we validate the shape of `story`? Otherwise there might be an error at this line: 
https://github.com/Codeminer42/cm42-central/pull/433/files#diff-e5085fcf0acb56251fd90be3f68fd0baR15

Reply:
Hi @matheusazzi thanks for the tip, I talked with @kostadriano and @talyssonoc and for while we'll keep this, because there are more than one place that are using it, after all we'll make a refactor and fix this and other stuffs!